### ISSUE TITLE
Please provide ImageNet pre-trained model for reproducibility

### ISSUE THREAD

**Issue by Hobeom** [lihaoxin05/egocentric-interaction#1]
Please provide a model that pre-trained 3000 steps in the human-to-human interaction part.

https://github.com/lihaoxin05/egocentric-interaction/blob/2d0258326619ef9dddb4dfb1736564d16deae557/human-human%20interaction/run.sh#L10

The model collapses in the training process of the JPL dataset. Please let me know the appropriate learning parameters.

**Comment by lihaoxin05** [comment]
The 'model.ckpt-3001' here is the model with trained attention module and motion module. You can use the one trained by myself in this link: https://drive.google.com/file/d/13HGJ25Zus6RtxLjyr27Zn1bLb6gs1Gn7/view?usp=sharing

**Comment by Hobeom** [comment]
Thank you for your quick response.

**Comment by Hobeom** [comment]
I trained on the JPL dataset using the ImageNet pre-learning model, but the accuracy needed higher. So, how do I adjust the loss_weight?

In my case, use this script.
```
python3 \
train.py \
--gpu_list 1 \
--dataset_dir data/JPL \
--dataset_list_file data/JPL/train_list.txt \
--checkpoint_dir ./checkpoints/ \
--visual_dir ./train_vis/ \
--vis_flag True \
--vis_freq 1000 \
--pretrain_model data/resume_int/model.ckpt \
--trainable_scopes interaction_net \
--learning_rate 1e-6 \
--max_steps 20000 \
--step_per_save 1000 \
--max_ckpt 20 \
--cls_loss_weight 1e0 \
--parsing_loss_weight 1e0 \
--pixel_loss_weight 1e0 \
--smooth_loss_weight 1e-3 \
--base_net_weight_reg 1e-4 \
--attention_net_weight_reg 1e-2 \
--motion_net_weight_reg 1e-2 \
--interaction_net_weight_reg 1e-4 \
--keep_prob 0.2 \
--batch_size 4 \
--num_class 7
```

I got an accuracy of 0.1316 on test_list, and even 0.1957 on train_list.
